packages/sdks/src/functions/evaluate/evaluate.ts

```diff
@@ -49,6 +49,7 @@ export function evaluate({
   const args: ExecutorArgs = {
     code: parseCode(code, { isExpression }),
     builder: getBuilderGlobals(),
+    Builder: getBuilderGlobals(),
```

Comment:
This is already handled here: https://github.com/BuilderIO/builder/blob/d391f3fe36137a579209a464c9cd5a79b8f7f67a/packages/sdks/src/functions/evaluate/helpers.ts#L34-L50

`getFunctionArguments` is called in each `evaluate`:

https://github.com/BuilderIO/builder/blob/d391f3fe36137a579209a464c9cd5a79b8f7f67a/packages/sdks/src/functions/evaluate/edge-runtime/edge-runtime.ts#L42-L47

So this change wouldn't make a difference

Reply:
ah i see, ty will keep digging in

Reply:
updated PR to only update the error message